packages/devtools-launchpad/package.json

```diff
@@ -3,13 +3,13 @@
   "version": "0.0.30",
   "license": "MPL-2.0",
   "repository": {
-    "url": "git://github.com/devtools-html/debugger.html.git",
+    "url": "git://github.com/devtools-html/devtools-core.git",
     "type": "git"
   },
   "bugs": {
-    "url": "https://github.com/devtools-html/debugger.html/issues"
+    "url": "https://github.com/devtools-html/devtools-core/issues"
   },
-  "homepage": "https://github.com/devtools-html/debugger.html#readme",
+  "homepage": "https://github.com/devtools-html/devtools-core#readme",
```

Comment:
Ah that was the intent, I keep mixing devtools-core and devtools-launchpad. thanks